gnuradio-runtime/include/gnuradio/math.h

```diff
@@ -43,8 +43,10 @@
 #define GR_M_LOG2E 1.4426950408889634074          /* log_2 e */
 #define GR_M_PI 3.14159265358979323846            /* pi */
 #define GR_M_PI_4 0.78539816339744830961566084582 /* pi/4 */
-#define GR_M_TWOPI (2 * GR_M_PI)                  /* 2*pi */
+#define GR_M_TWOPI 6.28318530717958647692         /* 2*pi */
 #define GR_M_SQRT2 1.41421356237309504880         /* sqrt(2) */
+#define GR_M_ONE_OVER_2PI 0.15915494309189533577  /* 1 / (2*pi) */
+#define GR_M_MINUS_TWO_PI -6.28318530717958647692 /* - 2*pi */
```

Comment:
No need for this. Just use ``-GR_M_TWOPI``.

Reply:
+1 from me too for `-GR_M_TWOPI`

Reply:
ping on this change

Reply:
I'm going ahead and will kind of veto the addition of constant `#define`s: these are worse than global scope variables, and any compiler in any optimizing step will just precompute `(2*GR_M_PI)` [at compile time](https://gcc.godbolt.org/z/Z8vrXz), even at `-O0`).

Reply:
... which probably means I will just clean up this PR  a bit :)

Reply:
Sounds good @marcusmueller.  Let me know if you need me to do anything.